clippy_lints/src/transmuting_null.rs

```diff
@@ -0,0 +1,96 @@
+use crate::consts::{constant_context, Constant};
+use crate::utils::{match_qpath, span_lint};
+use if_chain::if_chain;
+use rustc::hir::{Expr, ExprKind};
+use rustc::lint::{LateContext, LateLintPass, LintArray, LintPass};
+use rustc::{declare_tool_lint, lint_array};
+use syntax::ast::LitKind;
+
+declare_clippy_lint! {
+    pub TRANSMUTING_NULL,
+    correctness,
+    "transmutes from a null pointer to a reference, which is Undefined Behavior"
```

Comment:
```suggestion
    "transmutes from a null pointer to a reference, which is undefined behavior"
```

Reply:
What a keen eye! This is totally right :+1:
I checked the docs of other UB-preventing lints and the precedent in Clippy's docs is to write it in lowercase :D

Reply:
Corrected!